spec/errorSpec.js

```diff
@@ -102,4 +102,24 @@ describe('things that should fail', function() {
     expect(fakeDriver.getDecimalNumber()).toBeCloseTo(3.1);
     expect(fakeDriver.getDecimalNumber()).not.toBeCloseTo(3.14);
   });
+
+  describe('native promises', function() {
```

Comment:
Never mind. That code appeared to me more complicated than it actually is. Let's merge this. :+1: